docs/UserGuide.md

```diff
@@ -62,116 +62,68 @@ Jobby comes equipped with a [GUI]() interface that provides visual feedback to y
 Hello and welcoem to Jobby! We are delighted that you've chosen our platform to manage your internship application process!
 
 Before we begin, please ensure that you have viewed the following sections of the guide:
```

Comment:
Should include hints that these sections are located before the starter guide, so that the reader knows that if they read from top down they don't need to jump back up.

Reply:
I dont quite catch what u mean